homeassistant/components/media_player/strings.json

```
@@ -159,5 +159,178 @@
     "receiver": {
       "name": "Receiver"
     }
+  },
+  "services": {
+    "turn_on": {
+      "name": "Turn on",
+      "description": "Turns a media player power on."
+    },
+    "turn_off": {
+      "name": "Turn off",
+      "description": "Turns a media player power off."
```

```suggestion
      "description": "Turns off the power of the media player."
```
Is it really powering off? Or standby? Could it turn it back on when it is completely powered off?

---

Depends on the implementation. Most cases this is standby; however, I'm pretty sure that is what people expect nowadays when hitting the button on the remote (to turn off the tv = standby noways really).